frontend/src/helpers/grading-service/api-wrapper.ts

```diff
@@ -0,0 +1,59 @@
+import HttpStatusCode from "@/types/HttpStatusCode";
+import axios, { AxiosError } from "axios";
+
+const api = axios.create({
+  baseURL: process.env.GRADING_API_URL ?? "http://localhost:8080/grading/api",
+  timeout: 5000,
```

Comment:
When we add POST /feedback/generate endpoint here, we probably need to increase the timeout limit to maybe 10s? Not sure what value is reasonable yet, but I guess 10s will work better than 5s.